File: Python Programming/Fibonacci/fibonacci.py
def fibonacci(n):
    if n in {0, 1}:
        return n
    return fibonacci(n-1) + fibonacci(n-2)


print("Enter the amount of terms: ")

terms = int(input())

for i in range(terms):
    terms -= 1
    print(fibonacci(terms+1))

File: C++ Programming/Fibonacci/fibonacci.cpp
#include <iostream>
using namespace std;

int fibonacci(int n){
	if (n == 0 | n == 1){
		return n;
	}
	return fibonacci(n-1) + fibonacci(n-2);
}

int main()
{
	int terms;
	cout << "Enter the number of terms: ";
	cin >> terms;

	for (int i = 0; i < terms;){
		terms--;
		cout << fibonacci(terms) << endl;
	}
	return 0;
}